Sources/Commands/UserToolchain.swift

```diff
@@ -57,6 +60,9 @@ public struct UserToolchain: Toolchain {
         return swiftCompiler.parentDirectory.appending(component: "swift")
     }
 
+    /// Path to xctest utility.
+    let xctest: AbsolutePath?
```

Comment:
IDK if this should be optional. Technically, it is possible to use SwiftPM without `xctest` so maybe its fine.

Reply:
Yes, it's not there on e.g. Linux.

Reply:
oh ya, right.